### PR TITLE
PAYG Public Cloud Updates

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Two pages in the Admin Guide were missing from the nav. This PR adds them back, then continues to edit and reorg per feedback.

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Fixes https://github.com/SUSE/spacewalk/issues/12362

### CODE REVIEW

File: modules/installation/pages/pubcloud-setup.adoc [hunk 1]
```diff
@@ -2,17 +2,55 @@
...
+
 
+For instructions on registering {productname} to {scc}, see xref:installation:server-setup.adoc[].
+
+
+
+.Procedure: Activating the Public Cloud Module
 . On the {productname} Server, open the {yast} management tool, and navigate to menu:Software[Software Repositories].
 . Click btn:[Add] and select [guimenu]``Extensions and Modules from Registration Server``.
 . In the [guimenu]``Available extensions`` field, select [guimenu]``Public Cloud Module``.
```
Comment (keichwa) [inline modules/installation/pages/pubcloud-setup.adoc:56]: I think this isn't a "field" but a "list".

**Review by keichwa — APPROVED** [review]

File: modules/installation/pages/pubcloud-setup.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/installation/pages/pubcloud-setup.adoc:56]: Well, it's a drop-down box, I guess, but doesn't "field" cover everything?

**Review by jcayouette — APPROVED** [review]

**Comment by keichwa** [comment]
Lana Brindley <notifications@github.com> writes:

> Well, it's a drop-down box, I guess, but doesn't "field" cover everything? 

I think
https://documentation.suse.com/style/current/single-html/docu_styleguide/index.html
actually wants us to use:  :wink:

    drop-down box

and avoid:

    combination box, combo box, combobox, dropdown, drop-down, drop-down
    menu, drop-down list box, popover, pull-down menu

note:

    noun; GUI element with a list that can opened by clicking on it,
    whether combined with a text box or not; if list entries start
    actions, use menu instead

(I'm not sure whether our language group now thinks different about it.)

-- 
Karl Eichwalder

**Comment by Loquacity** [comment]
> Lana Brindley <notifications@github.com> writes:
> Well, it's a drop-down box, I guess, but doesn't "field" cover everything?
> I think https://documentation.suse.com/style/current/single-html/docu_styleguide/index.html actually wants us to use:  drop-down box 

That's all well and good, but all GUI elements that can accept user input are fields (including drop-down boxes). All drop-down boxes are fields, but not all fields are drop-down boxes.

**Pull request merged**